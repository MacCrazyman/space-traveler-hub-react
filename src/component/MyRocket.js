import React from 'react';
import PropTypes from 'prop-types';

const MyRocket = ({ rocketName }) => (
  <div>
    <p>{rocketName}</p>
  </div>
);

MyRocket.propTypes = {
  rocketName: PropTypes.string.isRequired,
};

export default MyRocket;
