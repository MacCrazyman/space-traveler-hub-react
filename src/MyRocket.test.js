import renderer from 'react-test-renderer';
import MyRocket from './component/MyRocket';

it('renders correctly', () => {
  const tree = renderer
    .create(<MyRocket rocketName="Rocket number 1" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
