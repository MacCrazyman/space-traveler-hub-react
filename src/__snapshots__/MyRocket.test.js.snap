// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="list-group-item"
  onClick={[Function]}
>
  <p>
    Rocket number 1
  </p>
</div>
`;
